client/Lykke.Service.Assets.Client/ContainerBuilderExtenions.cs

```diff
@@ -19,60 +17,48 @@ public static class ContainerBuilderExtenions
         /// Register the asset services.
         /// </summary>
         /// <param name="builder">The container builder for adding the services to.</param>
-        /// <param name="settings">The asset settings.</param>
-        /// <param name="autoRefresh">The marker of an expiring caches or a self refreshing cache for the assets and asset-pairs usage.</param>
+        /// <param name="serviceUrl">Service endpoint URL.</param>
+        /// <param name="registerDefaultAssetsReadModel">Whether to register the default in-memory assets and asset-pairs read model.
+        /// Please call IBoundedContextRegistration.WithAssetsReadModel during configuration of your bounded context.</param>
         [UsedImplicitly]
-        public static void RegisterAssetsClient(this ContainerBuilder builder, AssetServiceSettings settings,
-            bool autoRefresh = true)
+        public static void RegisterAssetsClient(this ContainerBuilder builder, string serviceUrl, bool registerDefaultAssetsReadModel = true)
```

Comment:
I would use IReloadingManager<AssetServiceSettings > as an argument

Reply:
But I wouldn't do that. This is an unnecessary dependence.